docs/pages/guides/linking.mdx

```diff
@@ -123,9 +126,12 @@ const redirectUrl = Linking.createURL('path/into/app', {
 
 This will resolve into the following, depending on the environment:
 
-- _Custom builds_: `myapp://path/into/app?hello=world`
+- _Builds and development builds_: `myapp://path/into/app?hello=world`
 - _Development in Expo Go_: `exp://127.0.0.1:8081/--/path/into/app?hello=world`. For SDK 48 and lower, the port number is `19000`.
-- _Published app in Expo Go_: `exp://u.expo.dev/[project-id]?channel-name=[channel-name]&runtime-version=[runtime-version]/--/path/into/app?hello=world`
+
+Again, when loading published updates in Expo Go this behavior is undefined, but may look something like the following:
```

Comment:
The section above defines the behavior. It would be more accurate to say here that the update group ID varies from update to update.